router.js

```diff
@@ -52,6 +53,39 @@ router.get(
   projectController.getActualSiteMixController
 )
 
+router.get(
+  '/home/projectStartDay/:id',
+  verifyToken,
+  authorizeMiddleware(['scrum_master', 'project_manager', 'resource_manager']),
+  homecontroller.getStartDayController
+)
+router.get(
+  '/home/projectCountEmployees/:id',
+  verifyToken,
+  authorizeMiddleware(['scrum_master', 'project_manager', 'resource_manager']),
+  homecontroller.getCountEmployeesControllers
+)
+
+router.get(
+  '/home/projectCountIterations/:id',
+  verifyToken,
+  authorizeMiddleware(['scrum_master', 'project_manager', 'resource_manager']),
+  homecontroller.getCountIterationsController
+)
+router.get(
+  '/home/projectCountScrums/:id',
+  verifyToken,
+  authorizeMiddleware(['scrum_master', 'project_manager', 'resource_manager']),
+  homecontroller.getCountScrumsController
+)
+router.get(
+  '/home/projectCountSites/:id',
+  verifyToken,
+  authorizeMiddleware(['scrum_master', 'project_manager', 'resource_manager']),
+  homecontroller.getCountSitesController
+)
+
 router.put('/Archive/:id', projectController.moveToArchive)
 
+router.get('*', (req, res) => res.send('not found'))
```

Comment:
Better to handle this in other way. maybe with proper status code...
we wrote this to test our code then. not needed any more.